Document/0x04e-Testing-Authentication-and-Session-Management.md

```diff
@@ -561,7 +561,7 @@ In addition, non-repudiation mechanisms should be applied to sensitive transacti
 In all cases, you should verify whether different devices are detected correctly. Therefore, the binding of the application to the actual device should be tested.
```

Comment:
Luckily this has a very pragmatic solution: we can remove the existing text and link to the Android and iOS test cases.

> Side note: we should get this text right but we don't need to put a lot of effort here since this test will surely disappear when the MASVS v2.0 is released (it changes the AUTH controls drastically and relies more on the ASVS for the remote endpoint topics).

```suggestion
In all cases, it should be possible for the user to log out specific open sessions and in some cases it might be interesting to fully block certain devices using a device identifier. See sections ["Testing Device Binding (Android)"](0x05j-Testing-Resiliency-Against-Reverse-Engineering.md#testing-device-binding-mstg-resilience-10) and ["Device Binding (iOS)"](0x06j-Testing-Resiliency-Against-Reverse-Engineering.md#device-binding-mstg-resilience-10) for further details.
```